src/Deployment/CliRunner.php

```diff
@@ -214,6 +215,20 @@ private function loadConfig()
 			? ['' => $config]
 			: array_filter($config, 'is_array');
 
+		if (isset($options['--section'])) {
+			$section = $options['--section'];
+			if (!strlen($section)) {
```

Comment:
The `strlen` check is required because the CLI accepts this kind of command:

```shell
php deployment --section= deployment.ini
# no section name!
```

Instead of raising an exception, the `section` option is filled with an empty string.